stdlib/public/runtime/ImageInspectionELF.h

```diff
@@ -33,6 +33,8 @@ struct SectionInfo {
 
 static constexpr const uintptr_t CurrentSectionMetadataVersion = 1;
 
+#define SWIFT_REFLECTION_METADATA_ELF_NOTE_MAGIC_STRING "swift_reflection_metadata_magic_string"
```

Comment:
Can you use a `constexpr const char []` instead?

Reply:
(no)(discussed offline)